app/src/main/res/layout-land/profile_edit_fragment.xml

```diff
@@ -35,11 +35,11 @@
 
       <de.hdodenhof.circleimageview.CircleImageView
         android:id="@+id/profile_edit_image"
-        android:layout_width="90dp"
-        android:layout_height="90dp"
-        android:layout_marginStart="28dp"
-        android:layout_marginTop="28dp"
-        android:elevation="2dp"
+        android:layout_width="@dimen/profile_edit_fragment_profile_width_and_height"
```

Comment:
Instead of one single dimens introduce two separate dimens as below:
profile_edit_fragment_profile_width
profile_edit_fragment_profile_height